packages/protocol/src/peer/ControllerCommissioner.ts

```diff
@@ -382,9 +382,6 @@ export class ControllerCommissioner {
             }
             throw e;
         }
-
-        await unsecureSession.destroy();
```

Comment:
Where the insecuresession is closed now? SHould that go into "finally"? (and when I see now ... yeahhh missing in error case before too)

Reply:
I also ended up adding a call to explicitly take the channel then added close here in a finally so it's less confusing